enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/KeywordNamedEntityNew.java

```diff
@@ -0,0 +1,23 @@
+package eu.europeana.enrichment.model.impl;
+
+import java.util.List;
+
+import dev.morphia.annotations.Embedded;
+
+@Embedded
+public class KeywordNamedEntityNew {
```

Comment:
this is not a keyword or a named entity, it is a position, please rename it to fit the purpose of the class

Reply:
I assume it is meant to be KeywordPossition